src/lib/collab/stack-client.js

```diff
@@ -114,6 +114,14 @@ export class ServiceClient {
     return this.stackClient.fetchJSON('POST', this.path(), doc)
   }
 
+  /**
+   * Force the note to be written to the io.cozy.files now
+   * @param {uuid} docId - note id
+   */
+  async sync(docId) {
+    await this.stackClient.fetchJSON('POST', this.path(docId, 'sync'))
```

Comment:
we should just return the promise, no?

Reply:
I am unsure about the benefit, but it won't harm anyway, 👍

Reply:
What would you expect and for what usage ?

Reply:
What is the benefit to await here? 

I think we'll do : 
`await client.sync()`

so 
`await await this.stackClient.fetchJSON('POST', this.path(docId, 'sync'))` brings nothing

and if we only do `client.sync()` it'll not wait the end of the call. So there is no benefits to await here, no?

Reply:
Either you `return this.stackClient.fetchJSON()` or you `await this.stackClient.fetchJSON()`.

Both will have the same async behaviour (if you wait `client.sync()`, it will wait the  `fetchJSON` from cozyClient). The difference is that the later does always return `undefined` and does not create a implicit API that someone may rely on. As the result of `fetchJSON` is useless to the caller, I would prefer not to return it.

Reply:
(I don't see any blocker anyways, so if it's important for you having this with a return, I will add a return)

Reply:
It's not a blocker for me neither. Your explanation is largely convincing ;)